src/features/canvass/components/GLCanvassMap/index.tsx

```diff
@@ -57,7 +57,7 @@ const GLCanvassMap: FC<Props> = ({ areas, assignment }) => {
       [180, 90],
     ];
 
-    const areaHoles = areas.map((area) => area.boundary.coordinates[0]);
+    const areaHoles = [selectedArea.boundary.coordinates[0]];
```

Comment:
I guess it's no longer necessary to use an array for this. Note how the array is just spread on line 64, so the contents of the array (a single item) could be moved there instead to simplify the code.

Reply:
Fixed by 8d1265ea9885b8e524935ab4ecac170e9809e3d8.